34329.8 <c-28#181> Created logger
34329.8 <c-28#181:26> Logger was updated
34329.8 <c-28#181:26> Started intializing cube lib
34329.8 <c-28#181:26> Finished intializing cube lib with worker
34329.8 <c-28#181:26> requestCubes => [31] Buffer size: 0
34329.8 <c-28#181:26> Sent requestCubes signal to root
34329.8 <c-28#181:26> Received 4 cubes from root
34329.8 <c-28#181:26> The main loop continues
34329.8 <c-28#181:26> Started solving a cube
34329.8 <c-28#181:26> Cube failed
34329.8 <c-28#181:26> Added failed cube
34329.8 <c-28#181:26> Started solving a cube
34329.9 <c-28#181:26> appl_pause was called
34329.9 <c-28#181:26> SUSPEND (0.01s since last cb)
34348.7 <c-28#181:26> appl_unpause was called
34348.7 <c-28#181:26> RESUME
34348.9 <c-28#181:26> appl_pause was called
34348.9 <c-28#181:26> SUSPEND (0.00s since last cb)
34519 <c-28#181:26> appl_unpause was called
34519 <c-28#181:26> RESUME
34541.4 <c-28#181:26> appl_pause was called
34541.4 <c-28#181:26> SUSPEND (0.00s since last cb)
34546.2 <c-28#181:26> appl_interrupt was called
34546.2 <c-28#181:26> RESUME
34546.2 <c-28#181:26> STOP after suspension
34546.2 <c-28#181:26> Solving interrupted
34546.2 <c-28#181:26> appl_interrupt was called
34546.2 <c-28#181:26> The main loop continues
34546.2 <c-28#181:26> Leaving the main loop due to an interruption
34546.2 <c-28#181:26> Started withdraw thread
34546.2 <c-28#181:26> Enter destructor of CubeWorker
34546.2 <c-28#181:26> Time waiting for messages: 0.000
34546.2 <c-28#181:26> Enter destructor of CubeWorkerInterface
34546.2 <c-28#181:26> Enter destructor of CubeRoot
34546.2 <c-28#181:26> Finished withdraw thread
34546.2 <c-28#181:26> appl_withdraw was called
34606.9 <c-28#181:26> Enter destructor
34606.9 <c-28#181:26> Time working: 22.434
34606.9 <c-28#181:26> Time suspended: 4.809
34606.9 <c-28#181:26> Joined cleanup thread
34606.9 <c-28#181:26> Exit destructor
34606.9 <c-28#181:26> Destructing logger
